application_integration/main.tf

```
@@ -0,0 +1,39 @@
+
+data "google_project" "test_project" {
```

```suggestion
data "google_project" "default" {
```

Use `default` as the local name unless required.